nixos/modules/misc/ids.nix

```diff
@@ -347,6 +347,7 @@ in
       #mailman = 316;  # removed 2019-08-30
       zigbee2mqtt = 317;
       # shadow = 318; # unused
+      nzbhydra2 = 319;
```

Comment:
Instead of this you should set `users.users.nzbhydra2.isSystemUser = true;`. See https://github.com/NixOS/rfcs/blob/master/rfcs/0052-dynamic-ids.md for a full explanation.

Reply:
Thanks! I didn't know.

From the RFC

> All unnecessary static ids can over time be removed from the central listing, replacing them with DynamicUser where applicable and/or dynamic ids allocated by NixOS.

A bit of an aside, but is this work ongoing? And are PRs accepted for this?

Reply:
I would assume. You could ping @Infinisil for more details.